# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/moz.build
# Commit: ee9744982673
# Full Hash: ee974498267381418f06efd6ab92c1f2c5f3735d
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-15 16:44:51
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r f5a4bfdaba40 -r ee9744982673 netwerk/base/moz.build
--- a/netwerk/base/moz.build	Tue Nov 15 05:34:40 2022 +0000
+++ b/netwerk/base/moz.build	Tue Nov 15 05:34:40 2022 +0000
@@ -247,7 +247,7 @@
 if CONFIG["FUZZING"]:
     SOURCES += [
         "FuzzyLayer.cpp",
-        "FuzzySecurityInfo.cpp",
+        "FuzzySocketControl.cpp",
     ]
 
 if CONFIG["FUZZING_INTERFACES"] and CONFIG["LIBFUZZER"]: